### PR TITLE
adding aria-disabled

### PR DESCRIPTION
**Pull request by samageloff**

### CODE REVIEW

**Comment by samageloff** [comment]
[(1) Add aria-disabled bool prop to Button](https://trello.com/c/HbM416d0/1066-1-add-aria-disabled-bool-prop-to-button)

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -55,7 +55,12 @@ const Button = (props) => {
   }
 
   return (
-    <button type={props.type} style={props.style} className={btnClasses} disabled={props.disabled || props.loading}
+    <button 
+    type={props.type}
```
Comment (will-hitchcock) [inline src/components/Button/Button.js]: Indent these

File: src/components/Button/Button.js [hunk 2]
```diff
@@ -82,6 +87,7 @@ Button.propTypes = {
    * Whether the button is disabled.
    */
   disabled: PropTypes.bool,
+  'aria-disabled': PropTypes.bool,
```
Comment (will-hitchcock) [inline src/components/Button/Button.js]: It doesn't look like we are accepting the `aria-disabled` prop on the `<Button />`, simply passing a value along to the native `<button />`. No need to add a prop type if we aren't accepting `aria-disabled`

Comment (samageloff) [reply src/components/Button/Button.js]: ⚡️

File: src/components/Button/Button.js (hunk 1 above)
Comment (samageloff) [reply src/components/Button/Button.js]: ⚡️

**Pull request merged**